tests/CallableTest.php

```diff
@@ -1327,6 +1327,11 @@ abstract class TestClass {
                         use TestTrait;
                     }'
             ],
+            'stringFunctionCall' => [
+                'code' => '<?php
+                    $bad_one = "hello";
+                    $a = $bad_one(1);',
```

Comment:
Shouldn't it complain about the function being unknown? I wouldn't keep this as a valid test.